webpack/assets/javascripts/react_app/redux/actions/bookmarks/bookmarks.fixtures.js

```diff
@@ -10,16 +28,20 @@ export const initialState = immutable({
 });
 
 export const requestData = { url: '/api/bookmarks', controller: 'hosts' };
+export const failRequestData = { url: '/api/bookmarks', controller: 'error' };
 
 const requestAction = {
   type: types.BOOKMARKS_REQUEST,
   payload: { controller: 'hosts' },
 };
 
 export const onFailureActions = [
-  requestAction,
   {
-    payload: { error: {}, item: { controller: 'hosts' } },
+    type: types.BOOKMARKS_REQUEST,
```

Comment:
Why not use the fixtures like other tests?

Reply:
shouldn't we use const types?

Reply:
you should, but if you look at the diff, than it was using it from the requestAction fixture.